src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserReleaseInviteRepositoryTests.cs

```diff
@@ -0,0 +1,306 @@
+﻿#nullable enable
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.DbUtils;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.ReleaseRole;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class UserReleaseInviteRepositoryTests
+    {
+        [Fact]
+        public async Task Create()
+        {
+            var releaseId = Guid.NewGuid();
+            var createdById = Guid.NewGuid();
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var repository = new UserReleaseInviteRepository(contentDbContext);
+                await repository.Create(
+                    releaseId: releaseId,
+                    email: "test@test.com",
+                    releaseRole: Contributor,
+                    emailSent: true,
+                    createdById: createdById);
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var userReleaseInvite = contentDbContext.UserReleaseInvites
+                    .AsQueryable()
+                    .SingleOrDefault();
```

Comment:
You could also `var userReleaseInvite = Assert.Single(contentDbContext.UserReleaseInvites);`

alternatively

```
await contentDbContext.UserReleaseInvites
                    .AsQueryable()
                    .SingleOrDefaultAsync();
```

for consistent use of async on database operations although I don't think it really matters.

I noticed it's not consistent throughout this class either, e.g. elsewhere there's a mixture of two different styles for enumerating all the table entries, e.g.

 `var remainingInvites = contentDbContext.UserReleaseInvites.ToList();`

vs.

```
                var userReleaseInvites = contentDbContext.UserReleaseInvites
                    .AsQueryable()
                    .ToList();
```

Reply:
Changed all calls to be async and added `AsQueryable` when appropriate